scripts/python-skywater-pdk/skywater_pdk/liberty.py

```diff
@@ -180,6 +194,36 @@ def top_corner_file(libname, corner, corner_type: TimingType, directory_prefix =
           corner_type=corner_type.file)
 
 
+def top_liberty_file(libname, corner, corner_type: TimingType, directory_prefix = "timing"):
+    """
+
+    >>> top_liberty_file("sky130_fd_sc_hd", "ff_100C_1v65", TimingType.ccsnoise)
+    'timing/sky130_fd_sc_hd__ff_100C_1v65_ccsnoise.lib'
+    >>> top_liberty_file("sky130_fd_sc_hd", "ff_100C_1v65", TimingType.basic)
+    'timing/sky130_fd_sc_hd__ff_100C_1v65.lib'
+    >>> top_liberty_file("sky130_fd_sc_hd", "ff_100C_1v65", TimingType.basic, "")
+    'sky130_fd_sc_hd__ff_100C_1v65.lib'
+
+    """
+    return top_corner_file(libname, corner, corner_type, directory_prefix).replace('.lib.json', '.lib')
```

Comment:
would it make sense to invert the relationship between the two function? that way `top_corner_file` could simply append `.json`.